CS3219-AY2223S1/cs3219-project-ay2223s1-g58
# Update nginx config to serve frontend built and reverse proxy to services

**What is the purpose of this pull request?**


**Overview of changes**
Fixes #75

- Change the original use of npm run start for the react app to serving the optimized build via nginx
- nginx also performs reverse proxy to user and matching service, other services can follow suit when required

**How it was tested/How to test**
- Run full stack using the command specified in root readme
- go to localhost:80
- check that user can still be logged in
- check that match service still works

**Service(s) Touched**
